src/config/field-overrides/programNotificationTemplate.js

```diff
@@ -129,6 +132,38 @@ const DataElementDropDown = compose(
     );
 });
 
+const NotificationRecipient = compose(
+    getContext({ d2: PropTypes.object }),
+    connect(
+        undefined,
+        boundOnUpdate
+    )
+)(({ onUpdate, d2, ...props }) => {
+    const label = d2.i18n.getTranslation('web_hook_url');
+    if (props.value === 'WEB_HOOK') {
+        return (
+            <div>
+                <Dropdown {...props} />
+                <TextField
+                    label={label}
+                    multiLine
```

Comment:
minor: should this be multiline...?

Reply:
Good catch. Most of this was copy pasted from the subject template component. Thanks!